packages/frontend/src/components/common/Filters/FilterInputs/dateUtils.ts

```diff
@@ -0,0 +1,45 @@
+import { WeekDay } from '@lightdash/common';
+import { DayOfWeek } from '@mantine/dates';
+import dayjs from 'dayjs';
+
+//
+// date inputs accepts start of the week day
+// prop is firstDayOfWeek: number 0-6
+// 0 – Sunday, 6 – Saturday, defaults to 1 – Monday
+// our internal WeekDay enum is a range from 0 (Monday) to 6 (Sunday)
+//
+export const convertWeekDayToDayOfWeek = (weekDay: WeekDay): DayOfWeek => {
```

Comment:
This is small, but the name of this function could be easier to understand. Right now I have no idea what it does without reading it. 

Maybe normalizeWeekDayIndex? Or something like that.

Reply:
great suggestion. called it `normalizeWeekDay` 👌